doc/qtcreator/src/mer/creator-developing-mer.qdoc

```diff
@@ -24,175 +24,83 @@
 
     \title Connecting Sailfish OS Devices
 
-    In Sailfish SDK terminology, a \e{target} contains everything that
-    you need to build applications for a particular type of Sailfish OS device,
-    such as a phone or a tablet. \QC fetches information about header files and
-    QML types from the target to be able to provide features such as syntax
-    highlighting and code completion.
+    You can connect Sailfish OS devices to the development PC to run, debug,
+    and analyze applications on from \QC.
 
-    The Sailfish SDK installation program installs targets on the
-    development host and configures the appropriate \l{glossary-buildandrun-kit}
-    {kit} for you in \QC. After the installation, you are ready to start
-    developing applications, as described in
-    \l{Creating a Sailfish OS Application}.
+    On Sailfish SDK side, various combinations of particular Sailfish OS
+    versions ran on particular HW architectures are supported with so called
```

Comment:
Maybe break this up a little bit to make it easier to read, e.g. "Various combinations of Sailfish OS versions and HW architectures are supported by so called SDK Build Targets. Each build target corresponds to a specific Sailfish OS version/HW architecture combination."

Reply:
Shortened by removing some words.

Note the important detail here which was more visible in the original text: This is a Qt Creator manual, not Sailfish SDK manual. From Qt Creator point of view, Sailfish SDK is an external, optional part. Compare with Android SDK etc. I did some cleanup in this respect, dropping the instructions like adding build engine and emulators manually - not because these are managed by the SDK installer but because these should be discovered automatically by QtC in the hypothetical scenario when upstream QtC integrates our plugin.

Reply:
And why I mention that: this is why the leading words "On Sailfish SDK side, " are important :)